compoments/radioFieldSet.js

```diff
@@ -177,6 +191,4 @@ RadioFieldSet.propTypes = {
   errorText: PropTypes.string,
   conditionalValue: PropTypes.object
 };
-export default RadioFieldSet;
-
-
+export default RadioFieldSet;
```

Comment:
Please restore the new line at the end of the file

Reply:
All done :-) .